### PR TITLE
CFE-2793: Bundle name is now part of the log message when aborting a bundle

### PR DESCRIPTION
**Pull request by olehermanse**
Ticket: https://tracker.mender.io/browse/CFE-2793

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom Please trigger a jenkins build

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com/job/pr-pipeline/738/)

https://ci.cfengine.com/job/pr-pipeline/738/

**Comment by olehermanse** [comment]
@cf-bottom Please trigger another jenkins build

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com/job/pr-pipeline/741/)

https://ci.cfengine.com/job/pr-pipeline/741/

File: libpromises/eval_context.c [hunk 1]
```diff
@@ -1605,7 +1605,16 @@ static bool EvalContextClassPut(EvalContext *ctx, const char *ns, const char *na
 
         if (IsRegexItemIn(ctx, ctx->heap_abort_current_bundle, context_copy))
         {
-            Log(LOG_LEVEL_ERR, "Bundle aborted on defined class '%s'", context_copy);
+            StackFrame *last_frame = LastStackFrameByType(ctx, STACK_FRAME_TYPE_BUNDLE);
```
Comment (vpodzime) [inline libpromises/eval_context.c]: You can use ``EvalContextStackCurrentBundle()`` here.

Comment (olehermanse) [reply libpromises/eval_context.c]: Fixed.

**Comment by olehermanse** [comment]
@cf-bottom Please trigger yet another jenkins build.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com/job/pr-pipeline/742/)

https://ci.cfengine.com/job/pr-pipeline/742/

**Pull request merged**